puppet/zulip/manifests/postgres_common.pp

```diff
@@ -46,6 +46,18 @@
       exec {'pip3_deps':
         command => 'python3 -m pip install pytz python-dateutil'
       }
+      file { '/usr/bin/pg_config':
+        ensure => 'link',
+        target => "/usr/pgsql-${zulip::base::postgres_version}/bin/pg_config"
+      }
+      file { '/usr/bin/pg_isready':
+        ensure => 'link',
+        target => "/usr/pgsql-${zulip::base::postgres_version}/bin/pg_isready"
+      }
+      file { '/usr/bin/psql':
+        ensure => 'link',
+        target => "/usr/pgsql-${zulip::base::postgres_version}/bin/psql"
+      }
```

Comment:
The `postgresql10` package from PGDG already installs a `/usr/bin/psql -> /etc/alternatives/pgsql-psql -> /usr/pgsql-10/bin/psql` link.

Reply:
In that case, in dev environment, should `psql` at https://github.com/zulip/zulip/blob/e2681372f3622dc8baa74cbbb3df7da5768f3f49/tools/lib/provision.py#L264 be removed? If there is actually `psql` in the path, why did I include it in the symlink list?